edbdeploy/data/terraform/aws/environments/ec2/outputs.tf

```diff
@@ -7,17 +7,27 @@ all:
 %{if var.pem_server["count"] > 0 ~}
     pemserver:
       hosts:
+%{if var.bdr_server["count"] > 0 ~}
+        pemserver1:
+%{else ~}
         pemserver1.${var.cluster_name}.internal:
+%{endif ~}
```

Comment:
The main reason is to differentiate the instances in the AWS console and other cloud consoles. If we use the same naming convention, it will be difficult to know which instance is part of BDR or primary standby.